src/App.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-app>
+    <Appbar />
+    <Drawer />
```

Comment:
A two way binding (cf [v-model](https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components) or [.sync](https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier)) will be less heavy than a whole store for just a menu that `App.vue` can handle + it's totally overkill...